src/main/java/org/craftercms/studio/api/v1/asset/Asset.java

```diff
@@ -32,16 +32,10 @@ public Asset(String repoPath, Path filePath) {
         this.filePath = filePath;
     }
 
-    /**
-     * Returns the repo path for the asset.
-     */
```

Comment:
Why remove comments?

Reply:
There is a recommendation not to document setters/getters and/or obvious stuff in order not to generate redundant or useless docs